### PR TITLE
IssueToken allow non-tradable materials

### PR DESCRIPTION
**Pull request by ipdae**

### CODE REVIEW

File: Lib9c/Model/Item/Inventory.cs [hunk 1]
```diff
@@ -442,7 +442,33 @@ public static bool IsMaterialRemovable(Item item, int id, long blockIndex)
...
+                if (material is TradableMaterial)
+                {
+                    return false;
+                }
+
+                return true;
+            }
+
+            return false;
+        }
```
Comment (moreal) [inline Lib9c/Model/Item/Inventory.cs:468]: It seems not to access any other methods, properties of `Inventory`. Shouldn't it be a static method and extracted from `Inventory` class?

File: Lib9c/Model/Item/Inventory.cs [hunk 2]
```diff
@@ -442,7 +442,33 @@ public static bool IsMaterialRemovable(Item item, int id, long blockIndex)
...
+        /// <param name="id">The ID of the material to remove.</param>
+        /// <returns>Returns a boolean indicating whether the item is removable.</returns>
+        public static bool IsMaterialRemovable(Item item, int id)
+        {
+            if (item.Locked)
+            {
+                return false;
+            }
+
+            if (item.item is Material material && material.Id == id)
```
Comment (moreal) [inline Lib9c/Model/Item/Inventory.cs:457]: `material.Id == id` seems not related to *'is removable'*. I think it should be like the below example because `id` is defined in `ItemBase`, not only for `Material`.

```csharp
List<Item> targetItems = _items.Where(item => item.Id == id && IsRemovableNonTradableMaterial(item)).ToList();
```

Or more..

```csharp
bool IsNonTradableMaterial(ItemBase item) => item is Material && item is not TradableMaterial;
List<Item> targetItems = _items.Where(item => !item.Locked && item.item.Id == id && IsNonTradableMaterial(item.item)).ToList();
```

**Review by moreal — COMMENTED** [review]
Just code review, not about bugs.

File: Lib9c/Model/Item/Inventory.cs (hunk 1 above)
Comment (ipdae) [reply Lib9c/Model/Item/Inventory.cs:468]: Since it is already defined as a static method, relocating it should not cause any significant issues.
